File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/til_sed/horisontal/HorisontalSedMapper.java
package no.nav.melosys.eessi.service.sed.mapper.til_sed.horisontal;

import no.nav.melosys.eessi.models.SedType;
import no.nav.melosys.eessi.service.sed.mapper.til_sed.SedMapper;

public class HorisontalSedMapper implements SedMapper {
    private final SedType sedType;

    public HorisontalSedMapper(SedType sedType) {
        this.sedType = sedType;
    }

    @Override
    public SedType getSedType() {
       return sedType;
    }
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/til_sed/horisontal/HorisontalSedMapper.kt
package no.nav.melosys.eessi.service.sed.mapper.til_sed.horisontal

import no.nav.melosys.eessi.models.SedType
import no.nav.melosys.eessi.service.sed.mapper.til_sed.SedMapper

class HorisontalSedMapper(private val sedType: SedType) : SedMapper {
    override fun getSedType(): SedType {
        return sedType
    }
}
